pkg/controller/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -39,7 +39,10 @@ func TestAddFinalizer(t *testing.T) {
 		mur := murtest.NewMasterUserRecord(t, "john")
 		memberClient := test.NewFakeClient(t)
 		hostClient := test.NewFakeClient(t, mur)
-		cntrl := newController(hostClient, s, newGetMemberCluster(true, v1.ConditionTrue),
+		config, err := configuration.LoadConfig(hostClient)
+		require.NoError(t, err)
```

Comment:
This could be moved into the `newController` function to not replicate it in all tests

Reply:
changed here: fa2316815d63d4a7db87100bc4a77cd7235df3ef

Reply:
and here: 4e7bbf729a5194bb03e1bcc11216c9d964be35de :facepalm: